src/Management/src/EndpointBase/Metrics/IMetricsObserverOptions.cs

```diff
@@ -19,6 +19,15 @@ public interface IMetricsObserverOptions
     /// </summary>
     string EgressIgnorePattern { get; }
 
+    /// <summary>
+    /// Gets Allow list of metrics that should be captured
+    /// </summary>
+    /// <remarks>
+    ///     Currently only applies to System.Runtime metrics captured by <see cref="EventCounterListener"/><para />
+    ///     See this list for values to choose from: <see href="https://docs.microsoft.com/dotnet/core/diagnostics/available-counters#systemruntime-counters" />
```

Comment:
```suggestion
    /// Gets Allow list of metrics that should be captured.
    /// </summary>
    /// <remarks>
    ///     Currently only applies to System.Runtime metrics captured by <see cref="EventCounterListener"/><para />
    ///     See this list for values to choose from: <see href="https://docs.microsoft.com/dotnet/core/diagnostics/available-counters#systemruntime-counters" />.
```

Reply:
None of the other elements have this - wouldn't be consistent. We can address resolving this in a separate PR.